.github/CONTRIBUTING.md

```diff
@@ -223,3 +217,43 @@ Provide some porting notes here if necessary.
 OpenZFS-issue: https://www.illumos.org/issues/1234
 OpenZFS-commit: https://github.com/openzfs/openzfs/commit/abcd1234
 ```
+
+#### Coverity Defect Fixes
+If you are submitting a fix to a Coverity defect, the commit message must
+meet the following guidelines:
+* The first line must be in the format of `Fix coverity defects: CID dddd, dddd...`
+where `dddd` represents each CID fixed by the commit.
+* The body of the commit message must be a list of Coverity defects and how
+each defect was corrected. Each defect must start in with a line in the format
```

Comment:
How about.

* The body of the commit message should list each Coverity defect and how it was corrected.

Reply:
Do you want me to leave the description of the format? Or should we just let the example fill in that part?